src/oscar/apps/analytics/abstract_models.py

```diff
@@ -101,11 +102,12 @@ class AbstractUserSearch(models.Model):
         on_delete=models.CASCADE,
         verbose_name=_("User"))
     query = models.CharField(_("Search term"), max_length=255, db_index=True)
-    date_created = models.DateTimeField(_("Date Created"), auto_now_add=True)
+    date_created = models.DateTimeField(_("Date Created"), auto_now_add=True, db_index=True)
```

Comment:
See [my previous comment](https://github.com/django-oscar/django-oscar/pull/3364#discussion_r426118842) - please remove this index.

Reply:
Should I change the ordering of this model to 'query' as well ?